packages/overture-schema-divisions-theme/src/overture/schema/divisions/division_area/models.py

```diff
@@ -70,6 +72,7 @@ class DivisionArea(Feature[Literal["divisions"], Literal["division_area"]], Name
         Field(
             description="Division ID of the division this area belongs to.",
         ),
+        RefersTo(referee=Division, relationship=Relationship.BELONGS_TO)
```

Comment:
Long term, `@belongsTo(Division)` is both more compact and reads as a sentence. (Where we could register the decorators according to the list of relationships.)

Reply:
One upside of the (updated based on our convo, but only incrementally) syntax `Reference(Relationship.BELONGS_TO, Division)` is that it makes it (IMO) more obvious that you are annotating a primitive value (of type `Id`) to show that it contains a reference, basically that it is a foreign key.

We're kind of conflating two concepts here: the foreign key reference to an ID, and the conceptual relationship that the foreign key is crudely modeling. The downside of the more compact `@belongs_to(Division)` syntax is that it reads like "this `Id` field belongs to division" rather than "this `Id` field is a reference modeling a belongs to relationship to a division"......................